docs/guide-bedtools.md

```diff
@@ -0,0 +1,141 @@
+---
+jupytext:
+  formats: md:myst
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.11.3
+kernelspec:
+  display_name: Python 3
+  language: python
+  name: python3
+---
+
+# Bioframe for bedtools users
+
+
+bioframe is built around the analysis of genomic intervals as a pandas [DataFrame](https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.html) in memory, rather than working with tab-delimited text files saved on disk.
+
+Bioframe supports reading a number of standard genomics text file formats via [`read_table`](https://bioframe.readthedocs.io/en/latest/api-fileops.html#bioframe.io.fileops.read_table), including BED files (see [schemas](https://github.com/open2c/bioframe/blob/main/bioframe/io/schemas.py)), which will load them as pandas DataFrames, a complete list of helper functions is [available here](API_fileops).
+
+For example, with gtf files, you do not need to turn them into bed files, you can directly read them into pandas (with e.g. [gtfparse](https://github.com/openvax/gtfparse/tree/master)).
+For gtfs, it is often convenient to rename the seqname column into chrom, the default column name used in bioframe.
+
+Any DataFrame object with `'chrom'`, `'start'`, and `'end'` columns will support the genomic [interval operations in bioframe](API_ops).
+
+Finally, if needed, bioframe provides a convenience function to write the back to a bed file using `to_bed`.
+
+
+## `bedtools intersect`
+
+### Original unique entries from the first bed `-u`
+
+Note that this gives one row per overlap and can contain duplicates,
+
+```sh
+bedtools intersect -u -a A.bed -b B.bed > out.bed
+```
+
+```py
+overlap = bf.overlap(A, B, how='inner', suffixes=('_1','_2'), return_index=True)
+out = A.loc[overlap['index_1'].unique()]
+```
+
+### Report the number of hits in B `-c`
+
+Reports 0 for A entries that have no overlap with B.
+
+```sh
+bedtools intersect -c -a A.bed -b B.bed > out.bed
+```
+
+```py
+out = bf.count_overlaps(A, B)
+```
+
+### Original entries from the first bed for each overlap`-wa`
+
+Note that this gives one row per overlap and can contain duplicates,
+
+```sh
+bedtools intersect -wa -a A.bed -b B.bed > out.bed
+```
+
+```py
+overlap = bf.overlap(A, B, how='inner', suffixes=('_1','_2'), return_index=True)
+out = A.loc[overlap['index_1']]
+# Alternatively
+out = A.loc[bioframe.ops._overlap_intidxs(A, B, how='inner')[:,0]]
```

Comment:
I should have chimed in here. This is not a public API so I'd leave it out until we find a better way to expose it.
```suggestion
```